apps/src/code-studio/initApp/loadApp.js

```diff
@@ -153,4 +337,84 @@ module.exports = function (callback) {
   } else {
     loadLastAttemptFromSessionStorage();
   }
+}
+
+window.dashboard = window.dashboard || {};
+window.dashboard.project = project;
+
+window.apps = {
```

Comment:
It's actually not a new global. This is just moved over from `initApp.js` which died a fiery death. But this global should be killed as well in the future.